package.json

```diff
@@ -46,6 +46,7 @@
     "globby": "^6.1.0",
     "jest": "^24.9.0",
     "jest-puppeteer": "^4.4.0",
+    "moment": "^2.25.3",
```

Comment:
I think this should be a dev dependency since it's just used be the e2e test.

Reply:
yeah! It's in `devDependencies` :smiley: